goodtablesio/blueprints/site.py

```diff
@@ -64,6 +64,11 @@ def source_s3_job(bucket, job):
     return _source('s3', bucket, job)
 
 
+@site.route('/settings')
+def settings():
```

Comment:
You'll need `@login_required` here (just noticed as I'm working on access checks)